builtin/mainmenu/tab_credits.lua

```diff
@@ -38,26 +38,26 @@ local active_contributors = {
 	"SmallJoker <mk939@ymail.com>",
 	"Lars Hofhansl <larsh@apache.org>",
 	"Jeija <jeija@mesecons.net>",
-	"Gregory Currie (gregorycu)",
-	"Sokomine <wegwerf@anarres.dyndns.org>",
 	"TeTpaAka",
 	"Jean-Patrick G (kilbith) <jeanpatrick.guerrero@gmail.com>",
 	"Diego Martínez (kaeza) <kaeza@users.sf.net>",
 	"Dániel Juhász (juhdanad) <juhdanad@gmail.com>",
 	"Rogier <rogier777@gmail.com>",
+	"red-001 <red-001@outlook.ie>",
+	"Vanessa Ezekowitz (VanessaE) <vanessaezekowitz@gmail.com>",
```

Comment:
Quite often there are PRs that need live testing before they are added to the engine. Often the person who does this is Vanessa, on her actual servers (not "test" servers). To me that is contributing and a valuable one. I don't think Vanessa cares if her name is in this section or not, but I believe it does belong here. Developing is about more than just coding.